runtime/ops/mod.rs

```diff
@@ -9,11 +9,16 @@ pub mod tty;
 pub mod web_worker;
 pub mod worker_host;
 
+use std::sync::Arc;
+
 use deno_core::OpState;
+use deno_features::FeatureChecker;
 
 /// Helper for checking unstable features. Used for sync ops.
 pub fn check_unstable(state: &OpState, feature: &str, api_name: &str) {
-  state.feature_checker.check_or_exit(feature, api_name);
+  state
+    .borrow::<Arc<FeatureChecker>>()
```

Comment:
I'm wondering if we should bury `Arc` inside `FeatureChecker`. Seems like an implementation detail. Feel free to do it in a follow up